config/config/package.json

```diff
@@ -27,7 +27,6 @@
   "scripts": {
     "prepublishOnly": "pnpm run compile",
     "lint": "eslint \"src/**/*.ts\" \"test/**/*.ts\"",
-    "test-with-preview": "ts-node test",
```

Comment:
There isn't a `test` script in this package. Running this tries to run `test/index.ts`, which doesn't work since it's a Jest test.  I think this is an unused script.